modbuslistener.cpp

```diff
@@ -155,6 +155,9 @@ void ModbusListener::readReady()
 
             uint32_t temp;
             temp = byte[0] + byte[1] + byte[2] + byte[3];
+// QA: are you sure about reinterpert_cast? why not dynamic cast? 
+// QA: as I understand, reinterpret_cast works on compiler level,
+// QA: and it may be dangerous 
```

Comment:
Thank you again. I'll check casts. It doesn't matter but here it depends on hardware.